PyInstaller/hooks/hook-_tkinter.py

```diff
@@ -137,6 +190,10 @@ def _collect_tkfiles(mod):
 
     tcltree = Tree(tcl_root, os.path.join('_MEI', tcldir),
                    excludes=['demos', '*.lib', 'tclConfig.sh'])
+
+    # handle workaround for ActiveTcl on OS X
+    _handle_activetcl_install(tcl_root, tcltree)
```

Comment:
Please wrap this in  `if is_darwin`, to make clear this is only for darwin. Thanks.